### PR TITLE
Remove unittest2 imports

### PR DESCRIPTION
**Pull request by souravsingh**

### CODE REVIEW

**Comment by souravsingh** [comment]
This is related to issue #3225 regarding running tests in conda-forge

File: gensim/test/test_ldamallet_wrapper.py [hunk 1]
```diff
@@ -10,7 +10,13 @@
 
 
 import logging
-import unittest2 as unittest
+import sys
+
+if sys.version_info < (2, 7):
```
Comment (menshikh-iv) [inline gensim/test/test_ldamallet_wrapper.py]: @souravsingh now we don't support Python version that older than 2.7, you no need this `if`

**Comment by souravsingh** [comment]
@menshikh-iv Can you take a look?

**Comment by menshikh-iv** [comment]
@souravsingh yes, we should to replace `unittest2` to `unittest` and remove it from env.

**Comment by menshikh-iv** [comment]
Thanks @souravsingh :+1:

**Pull request merged**